### PR TITLE
Updated udunits2 to v2.2.20

### PR DESCRIPTION
**Pull request by ocefpaf**
I am not 100% sure about the license.  (Actually [I am 83%](http://www.what-license.com/) sure :wink:)

### CODE REVIEW

**Comment by QuLogic** [comment]
Hmm, Fedora says MIT...

File: udunits2/meta.yaml [hunk 1]
```diff
@@ -1,11 +1,11 @@
 package:
     name: udunits2
-    version: 2.2.17
+    version: 2.2.20
```
Comment (rhattersley) [inline udunits2/meta.yaml]: So is this version 2.2.20 of udunits-2? Or is it version 2.2.20 of udunits? :stuck_out_tongue_winking_eye:

Comment (ocefpaf) [inline udunits2/meta.yaml]: We can call it udunits2.2 v20 if you wish :smiley_cat:

**Comment by ocefpaf** [comment]
> Hmm, Fedora says MIT...

... Maybe we can just add a link to the license file instead.  I believe it is important to add the license information in the yaml file.

**Comment by QuLogic** [comment]
Wow, that website really said GPL? [This](https://github.com/Unidata/UDUNITS-2/blob/master/COPYRIGHT) is in no way, shape, or form even close to a GPL license.

**Comment by ocefpaf** [comment]
Thanks @QuLogic.  I will stop using that site!

File: udunits2/meta.yaml (hunk 1 above)
Comment (rhattersley) [inline udunits2/meta.yaml]: :+1: :grin:

**Comment by QuLogic** [comment]
There's one dead giveaway for GPL licenses... They've got their own name in the text.

**Comment by pelson** [comment]
This just needs a rebase and I think it will pass.

**Comment by ocefpaf** [comment]
Rebase. But AppVeyor's log are not fun to read :unamused:

**Comment by ocefpaf** [comment]
Rebased again and I think AppVeyor is happy now with latest conda.

**Pull request merged**